electronics/imu-selection.md

```diff
@@ -0,0 +1,54 @@
+---
+parent: Electronics
+---
+
+# _How to choose an IMU_
+
+-   Don't consider price as your main deciding factor, the cheapest option might
+    still have the worst price/performance.- Consider expected G loads, then
+    double or triple for safety margins.
+-   Lower drift figures are usually better but be wary of incorrectly reported
+    measurements.
+-   Bosch and ST have FAR better sensing topology than Invensense almost as a rule
+-   Note that the BNO055 has onboard sensor processing you pay extra for that
+    will NOT work in a rocket context, buy the BMX055.
+-   Spending that bit extra on a better sensor will help alleviate pain down the
+    line as you don't have to fight it as much
+-   Consider ditching breakout boards, instead design your own integration on
+    board, follow the datasheet recommended schematic and layout.
+-   Breakouts being attached to your board can lead to sensor decoupling sensed
+    motion from your airframe due to waffling around which is why we don't really
+    recommend using them even though it seems easier. SMD soldering is a useful
+    skill and opens the door to much more accurate sensor implementations.
+
+## IMU Suggestions
+
+**DO NOT USE THE MPU SERIES**. Their sensing technology is ancient, drifts more
+than Initial D, has incorrect datasheet representation, and useless DMP.
+
+Supply your sensors with a clean, quiet, and correct voltage reference for best
+results
+
+**A** = Accelerometer
+**G** = Gyroscope
+**M** = Magnetometer
+
+| Name      | Price    | [DOF]     | Ranges                   | Precision           | Drift figures                 | Library? | Recommended                                                                                    |
+| --------- | -------- | --------- | ------------------------ | ------------------- | ----------------------------- | -------- | ---------------------------------------------------------------------------------------------- |
+| MPU6050   | $2 (EOL) | 6 (A+G)   | ±16g ±2000°/sec          | 16Bit               | ±50mg(up to ±80mg) ±20º/s     | Many     | Bad drift, bad lifetime, **do not use**                                                        |
```

Comment:
Doesn't have to be a part of this PR, but the "Drift figures" section is very hard to read due to all figures being on the same line and certain sensors not having all of them. 
Sources should be provided for these as well.